api/src/uvisor-input.S

```diff
@@ -107,6 +109,10 @@ __uvisor_page_size:
     /* uvisor default page size is 16kB - user can override weak reference */
     .long 16384
 
+__uvisor_box_index_size:
+    /* default index size is 64B - user can override weak reference */
+    .long 64
```

Comment:
Consider removing this overridable definition, as per https://github.com/ARMmbed/uvisor/pull/230#discussion_r65719493